doc/how_to/custom_components/reactive_html/reactive_html_layout.md

```diff
@@ -0,0 +1,292 @@
+# Create Layouts With ReactiveHTML
+
+In this guide you will learn how to build custom layouts using HTML and `ReactiveHTML`.
+
+If you are not familiar with HTML then the [W3 HTML School](https://www.w3schools.com/html/default.asp)
+is a good resource to learn from. You can also ask ChatGPT for help. It can often provide you with
+a good HTML starting point that you can fine tune.
+
+## A layout of a single object
+
+You can layout a single object as follows.
+
+```{pyodide}
+import panel as pn
+import param
+
+pn.extension()
+
+class LayoutSingleObject(pn.reactive.ReactiveHTML):
+    object = param.Parameter()
+
+    _ignored_refs = ("object",)
+
+    _template = """
+    <div style="height:100%;width:100%">
```

Comment:
Why is it that I always have to add a wrapper div and set the `height` and `width` to 100% @philippjfr . And why do I have to explain users this as well. This seems like friction. Could it be done automatically?